src/mpi/datatype/typerep/src/typerep_yaksa_pack.c

```diff
@@ -15,6 +15,11 @@ int MPIR_Typerep_copy(void *outbuf, const void *inbuf, MPI_Aint num_bytes)
     int mpi_errno = MPI_SUCCESS;
     int rc;
 
+    if (outbuf == NULL || inbuf == NULL) {
```

Comment:
Could this branch just be for `num_bytes == 0`? That way it skips all the logic below even when valid buffers are supplied.

Reply:
Yes, of course!